Add Ecto formatter config example to "Getting Started" guide (#4612)
elixir-ecto/ecto: guides/introduction/Getting Started.md

querying at all.

That's the first two steps taken now. We have installed Ecto and Postgrex as

> dependencies of our application.
> 
> Now, we can change our `.formatter.exs` file so that Ecto's custom formatter
> rules will be applied when we run `mix format`:

Reviewer: ```suggestion
dependencies of our application. Next let's update the `.formatter.exs`
file so that Ecto's rules will be applied on `mix format`:
```

```elixir
[
  # Add these lines to enable Ecto formatter rules
  import_deps: [:ecto, :ecto_sql],

>   subdirectories: ["priv/*/migrations"],

Author: OK, done.

In theory, it would be better with more info. But this at least gets the reader going with the idea that there are formatter rules, without cluttering up the intro guide too much.
